records_mover/db/postgres/copy_options/__init__.py

```diff
@@ -34,20 +37,59 @@ def needs_csv_format(hints: RecordsHints) -> bool:
     return False
 
 
-def postgres_copy_options(unhandled_hints: Set[str],
-                          load_plan: RecordsLoadPlan) -> Tuple[Optional[DateInputStyle],
-                                                               PostgresCopyOptions]:
+# unloading
+def postgres_copy_to_options(unhandled_hints: Set[str],
+                             delimited_records_format: DelimitedRecordsFormat,
+                             fail_if_cant_handle_hint: bool) ->\
+                                         Tuple[DateOutputStyle,
+                                               Optional[DateOrderStyle],
+                                               PostgresCopyOptions]:
+    hints = delimited_records_format.hints
+
+    if needs_csv_format(hints):
+        copy_options = postgres_copy_options_csv(unhandled_hints,
+                                                 hints,
+                                                 fail_if_cant_handle_hint,
+                                                 CopyOptionsMode.UNLOADING)
+    else:
+        copy_options = postgres_copy_options_text(unhandled_hints,
+                                                  hints,
+                                                  fail_if_cant_handle_hint,
+                                                  CopyOptionsMode.UNLOADING)
+
+    date_output_style, date_order_style =\
+        determine_output_date_order_style(unhandled_hints,
+                                          hints,
+                                          fail_if_cant_handle_hint)
```

Comment:
The semantics of the date-related commands is fundamentally different, though - Postgres is very limited on date output, but very forgiving on date input, so I broke the handling of them out into two separate input/output functions.